scripts/check_formatting

```diff
@@ -20,3 +20,8 @@ find scripts -type f -exec shellcheck {} +
 # Fortunately, rustfmt has the --check option that will make it exit with 1
 # if formatting has to be applied.
 find examples rust -type f -name '*.rs' -exec rustfmt --check {} +
+
+# Check embedded code snippets are up-to-date.
+grep --recursive --files-with-matches embedmd docs | while IFS= read -r file; do
```

Comment:
Also apply the corresponding fix in the `format` script?

Reply:
Ah, I hadn't noticed there was a `format` script &ndash; done.